[CI] ShardBulkInferenceActionFilterIT testBulkOperations {useLegacyFormat=false useSyntheticSource=false} failing

**Build Scans:**
- [elasticsearch-intake #38134 / part5](https://gradle-enterprise.elastic.co/s/6jwfmdlnupwz2)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:inference:internalClusterTest" --tests "org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT.testBulkOperations {useLegacyFormat=false useSyntheticSource=false}" -Dtests.seed=5165F44D7B673EE4 -Dtests.locale=en-IE -Dtests.timezone=Asia/Ashkhabad -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testBulkOperations%20%7BuseLegacyFormat%3Dfalse%20useSyntheticSource%3Dfalse%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.Exception: Test abandoned because suite timeout was reached.
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testBulkOperations {useLegacyFormat=false useSyntheticSource=false}
- [main] 1 consecutive failures in step part5
- [main] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

I did a brief review of the logs for https://gradle-enterprise.elastic.co/s/pjqh2dzfg36mu , and yikes, sometimes we hit something horrible:
```
 <testcase name="testItemFailures {useLegacyFormat=true useSyntheticSource=true}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="18.441"/>
  <testcase name="testBulkOperations {useLegacyFormat=true useSyntheticSource=true}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="8.816"/>
  <testcase name="testRestart {useLegacyFormat=true useSyntheticSource=true}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="12.396"/>
  <testcase name="testItemFailures {useLegacyFormat=false useSyntheticSource=true}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="1.408"/>
  <testcase name="testBulkOperations {useLegacyFormat=false useSyntheticSource=true}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="7.235"/>
  <testcase name="testRestart {useLegacyFormat=false useSyntheticSource=true}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="9.76"/>
  <testcase name="testItemFailures {useLegacyFormat=false useSyntheticSource=false}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="1.76"/>
  <testcase name="testRestart {useLegacyFormat=false useSyntheticSource=false}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="8.172"/>
  <testcase name="testItemFailures {useLegacyFormat=true useSyntheticSource=false}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="2.889"/>
  <testcase name="testBulkOperations {useLegacyFormat=true useSyntheticSource=false}" classname="org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT" time="1128.425">
```

See the insane time this takes.

For this failure, we hit something like this:
```
[2026-04-22T11:59:51,503][WARN ][o.e.e.NodeEnvironment    ][suite] lock assertion failed
java.nio.channels.ClosedByInterruptException: null
	at java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:214) ~[?:?]
	at sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:182) ~[?:?]
	at sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:505) ~[?:?]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.store.NativeFSLockFactory$NativeFSLock.ensureValid(NativeFSLockFactory.java:169) ~[lucene-core-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.elasticsearch.env.NodeEnvironment.assertEnvIsLocked(NodeEnvironment.java:1287) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.env.NodeEnvironment.nodeDataPaths(NodeEnvironment.java:1046) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster$NodeAndClient.markNodeDataDirsAsPendingForWipe(InternalTestCluster.java:1159) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster$NodeAndClient.close(InternalTestCluster.java:1143) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:71) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:119) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster.close(InternalTestCluster.java:987) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.CloseableTestClusterWrapper.close(CloseableTestClusterWrapper.java:27) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:71) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:119) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.ESIntegTestCase.clearClusters(ESIntegTestCase.java:599) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.ESIntegTestCase.afterClass(ESIntegTestCase.java:2982) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:565) ~[?:?]
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1763) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:909) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.elasticsearch.entitlement.bootstrap.TestEntitlementsRule$1.evaluate(TestEntitlementsRule.java:78) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.cluster.metadata.TemplateDecoratorRule$1.evaluate(TemplateDecoratorRule.java:48) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.junit.rules.RunRules.evaluate(RunRules.java:20) ~[junit-4.13.2.jar:4.13.2]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850) ~[randomizedtesting-runner-2.8.2.jar:?]
	at java.lang.Thread.run(Thread.java:1516) ~[?:?]
[2026-04-22T11:59:51,505][WARN ][o.e.e.NodeEnvironment    ][suite] lock assertion failed
java.nio.channels.ClosedByInterruptException: null
	at java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:214) ~[?:?]
	at sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:182) ~[?:?]
	at sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:505) ~[?:?]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.store.NativeFSLockFactory$NativeFSLock.ensureValid(NativeFSLockFactory.java:169) ~[lucene-core-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.elasticsearch.env.NodeEnvironment.assertEnvIsLocked(NodeEnvironment.java:1287) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.env.NodeEnvironment.nodeDataPaths(NodeEnvironment.java:1046) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster$NodeAndClient.markNodeDataDirsAsPendingForWipe(InternalTestCluster.java:1159) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster$NodeAndClient.close(InternalTestCluster.java:1143) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:71) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:119) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster.close(InternalTestCluster.java:987) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.CloseableTestClusterWrapper.close(CloseableTestClusterWrapper.java:27) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:71) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:119) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.ESIntegTestCase.clearClusters(ESIntegTestCase.java:599) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.ESIntegTestCase.afterClass(ESIntegTestCase.java:2982) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:565) ~[?:?]
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1763) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:909) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.elasticsearch.entitlement.bootstrap.TestEntitlementsRule$1.evaluate(TestEntitlementsRule.java:78) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.cluster.metadata.TemplateDecoratorRule$1.evaluate(TemplateDecoratorRule.java:48) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.junit.rules.RunRules.evaluate(RunRules.java:20) ~[junit-4.13.2.jar:4.13.2]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850) ~[randomizedtesting-runner-2.8.2.jar:?]
	at java.lang.Thread.run(Thread.java:1516) ~[?:?]
[2026-04-22T11:59:51,505][WARN ][o.e.e.NodeEnvironment    ][suite] lock assertion failed
java.nio.channels.ClosedByInterruptException: null
	at java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:214) ~[?:?]
	at sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:182) ~[?:?]
	at sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:505) ~[?:?]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.mockfile.FilterFileChannel.size(FilterFileChannel.java:82) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.store.NativeFSLockFactory$NativeFSLock.ensureValid(NativeFSLockFactory.java:169) ~[lucene-core-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.elasticsearch.env.NodeEnvironment.assertEnvIsLocked(NodeEnvironment.java:1287) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.env.NodeEnvironment.nodeDataPaths(NodeEnvironment.java:1046) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster$NodeAndClient.markNodeDataDirsAsPendingForWipe(InternalTestCluster.java:1159) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster$NodeAndClient.close(InternalTestCluster.java:1143) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:71) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:119) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.InternalTestCluster.close(InternalTestCluster.java:987) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.CloseableTestClusterWrapper.close(CloseableTestClusterWrapper.java:27) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:71) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.core.IOUtils.close(IOUtils.java:119) ~[elasticsearch-core-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.ESIntegTestCase.clearClusters(ESIntegTestCase.java:599) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.test.ESIntegTestCase.afterClass(ESIntegTestCase.java:2982) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:565) ~[?:?]
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1763) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:909) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.elasticsearch.entitlement.bootstrap.TestEntitlementsRule$1.evaluate(TestEntitlementsRule.java:78) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.cluster.metadata.TemplateDecoratorRule$1.evaluate(TemplateDecoratorRule.java:48) ~[framework-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47) ~[lucene-test-framework-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.junit.rules.RunRules.evaluate(RunRules.java:20) ~[junit-4.13.2.jar:4.13.2]
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390) ~[randomizedtesting-runner-2.8.2.jar:?]
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850) ~[randomizedtesting-runner-2.8.2.jar:?]
	at java.lang.Thread.run(Thread.java:1516) ~[?:?]
[2026-04-22T11:59:52,829][WARN ][o.e.c.InternalClusterInfoService][node_s2][management][T#1] failed to retrieve stats for node [QSJrM0xUSs--Ae7rwJVJbA]
org.elasticsearch.transport.RemoteTransportException: [node_s2][127.0.0.1:20171][cluster:monitor/nodes/stats[n]]
Caused by: org.apache.lucene.store.AlreadyClosedException: FileLock invalidated by an external force: NativeFSLock(path=/dev/shm/bk/bk-agent-prod-gcp-1776871395300710050/elastic/elasticsearch-intake/x-pack/plugin/inference/build/testrun/internalClusterTest/nodesTemp/org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT_4042188A163A360D-001/tempDir-002/node_s2/node.lock,impl=sun.nio.ch.FileLockImpl[0:9223372036854775807 exclusive invalid],creationTime=2026-04-22T15:39:56.74286321Z)
	at org.apache.lucene.store.NativeFSLockFactory$NativeFSLock.ensureValid(NativeFSLockFactory.java:165) ~[lucene-core-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.elasticsearch.env.NodeEnvironment.assertEnvIsLocked(NodeEnvironment.java:1287) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.env.NodeEnvironment.dataPaths(NodeEnvironment.java:1083) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsProbe.stats(FsProbe.java:47) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService$FsInfoCache.refresh(FsService.java:94) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService$FsInfoCache.refresh(FsService.java:80) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.common.util.SingleObjectCache.getOrRefresh(SingleObjectCache.java:43) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService.lambda$new$1(FsService.java:58) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService.stats(FsService.java:68) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.node.NodeService.stats(NodeService.java:191) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.admin.cluster.node.stats.TransportNodesStatsAction.nodeOperation(TransportNodesStatsAction.java:160) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.admin.cluster.node.stats.TransportNodesStatsAction.nodeOperation(TransportNodesStatsAction.java:46) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperationAsync(TransportNodesAction.java:257) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.lambda$messageReceived$0(TransportNodesAction.java:273) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.ActionListener.run(ActionListener.java:490) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:271) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:86) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.transport.TransportService$6.doRun(TransportService.java:1126) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1114) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614) ~[?:?]
	at java.lang.Thread.run(Thread.java:1516) ~[?:?]
[2026-04-22T11:59:52,830][WARN ][o.e.c.InternalClusterInfoService][node_s2][management][T#1] failed to retrieve stats for node [M5TVThDyQ5WxdqCHyCX-dw]
org.elasticsearch.transport.RemoteTransportException: [node_s0][127.0.0.1:20173][cluster:monitor/nodes/stats[n]]
Caused by: org.apache.lucene.store.AlreadyClosedException: FileLock invalidated by an external force: NativeFSLock(path=/dev/shm/bk/bk-agent-prod-gcp-1776871395300710050/elastic/elasticsearch-intake/x-pack/plugin/inference/build/testrun/internalClusterTest/nodesTemp/org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT_4042188A163A360D-001/tempDir-002/node_s0/node.lock,impl=sun.nio.ch.FileLockImpl[0:9223372036854775807 exclusive invalid],creationTime=2026-04-22T15:39:48.777145701Z)
	at org.apache.lucene.store.NativeFSLockFactory$NativeFSLock.ensureValid(NativeFSLockFactory.java:165) ~[lucene-core-10.4.0.jar:10.4.0 9983b7ce7fdd04f4d357688fb85c14277c15ea8d - 2026-02-20 18:05:31]
	at org.elasticsearch.env.NodeEnvironment.assertEnvIsLocked(NodeEnvironment.java:1287) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.env.NodeEnvironment.dataPaths(NodeEnvironment.java:1083) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsProbe.stats(FsProbe.java:47) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService$FsInfoCache.refresh(FsService.java:94) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService$FsInfoCache.refresh(FsService.java:80) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.common.util.SingleObjectCache.getOrRefresh(SingleObjectCache.java:43) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService.lambda$new$1(FsService.java:58) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.monitor.fs.FsService.stats(FsService.java:68) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.node.NodeService.stats(NodeService.java:191) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.admin.cluster.node.stats.TransportNodesStatsAction.nodeOperation(TransportNodesStatsAction.java:160) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.admin.cluster.node.stats.TransportNodesStatsAction.nodeOperation(TransportNodesStatsAction.java:46) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperationAsync(TransportNodesAction.java:257) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.lambda$messageReceived$0(TransportNodesAction.java:273) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.ActionListener.run(ActionListener.java:490) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:271) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:86) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.transport.InboundHandler.doHandleRequest(InboundHandler.java:319) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.transport.InboundHandler$1.doRun(InboundHandler.java:331) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1114) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27) ~[elasticsearch-9.5.0-SNAPSHOT.jar:9.5.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614) ~[?:?]
	at java.lang.Thread.run(Thread.java:1516) ~[?:?]
```

The test is hitting [an assertion in TransportShardBulkAction](https://github.com/Mikep86/elasticsearch/blob/30f603e45847ceab6a5872f48843f1259a8b108a/server/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java#L855-L855):

```
java.lang.AssertionError: Unexpected request operation type on replica: update {[test-index][IZLUgAoQABBMPbtkjyDt], doc_as_upsert[false], doc[index {[test-index][IZLUgAoQABBMPbtkjyDt], source[{"dense_field":{"text":"bJLLiuGagfx","inference":{"inference_id":"dense-endpoint","model_settings":{"service":"text_embedding_test_service","task_type":"text_embedding","dimensions":68,"similarity":"l2_norm","element_type":"float"},"chunks":[{"text":"bJLLiuGagfx","embeddings":[45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0,49.0,55.0,45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0,49.0,55.0,45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0,49.0,55.0,45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0,49.0,55.0,45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0,49.0,55.0,45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0,49.0,55.0,45.0,49.0,54.0,53.0,56.0,55.0,57.0,51.0]}]}},"sparse_field":{"text":"ZZNanAZoPknTzrqFW","inference":{"inference_id":"sparse-endpoint","model_settings":{"service":"test_service","task_type":"sparse_embedding"},"chunks":[{"text":"ZZNanAZoPknTzrqFW","embeddings":{"feature_0":1.6984435E9,"feature_1":1.6984435E9,"feature_2":1.6984435E9,"feature_3":1.6984435E9,"feature_4":1.6984435E9}}]}}}]}], scripted_upsert[false], detect_noop[true]};primary result: UpdateResponse[index=test-index,id=IZLUgAoQABBMPbtkjyDt,version=2,seqNo=1,primaryTerm=1,result=updated,shards=ShardInfo{total=0, successful=0, failures=[]}]
	at __randomizedtesting.SeedInfo.seed([4042188A163A360D]:0)
	at org.elasticsearch.action.bulk.TransportShardBulkAction.performOpOnReplica(TransportShardBulkAction.java:855)
	at org.elasticsearch.action.bulk.TransportShardBulkAction.performOnReplica(TransportShardBulkAction.java:818)
	at org.elasticsearch.action.bulk.TransportShardBulkAction.performOnReplica(TransportShardBulkAction.java:782)
	at org.elasticsearch.action.bulk.TransportShardBulkAction.lambda$dispatchedShardOperationOnReplica$8(TransportShardBulkAction.java:760)
	at org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:392)
	at org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnReplica(TransportShardBulkAction.java:745)
	at org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnReplica(TransportShardBulkAction.java:80)
	at org.elasticsearch.action.support.replication.TransportWriteAction$2.doRun(TransportWriteAction.java:272)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:35)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1114)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
	at java.base/java.lang.Thread.run(Thread.java:1516)
```

An update operation is ending up on a replica

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -288,18 +288,21 @@ tests:
 - class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
   method: testReplaceChildren {class org.elasticsearch.xpack.esql.plan.physical.ExchangeSinkExec}
   issue: https://github.com/elastic/elasticsearch/issues/147164
-- class: org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT
-  method: testBulkOperations {useLegacyFormat=true useSyntheticSource=true}
-  issue: https://github.com/elastic/elasticsearch/issues/147169
 - class: org.elasticsearch.xpack.prometheus.PrometheusMetadataRestIT
   method: testMultipleEntriesForSameMetricAcrossStreams
   issue: https://github.com/elastic/elasticsearch/issues/147170
 - class: org.elasticsearch.xpack.security.operator.OperatorPrivilegesIT
   method: testNonOperatorUserCanCallAnalyzeRepositoryAPI
   issue: https://github.com/elastic/elasticsearch/issues/147171
-- class: org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT
-  method: testBulkOperations {useLegacyFormat=false useSyntheticSource=true}
-  issue: https://github.com/elastic/elasticsearch/issues/147177
+- class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
+  method: testReplaceChildren {class org.elasticsearch.xpack.esql.plan.logical.promql.operator.VectorBinaryComparison}
+  issue: https://github.com/elastic/elasticsearch/issues/147195
+- class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
+  method: testTransform {class org.elasticsearch.xpack.esql.plan.logical.promql.UnresolvedPromqlFunction}
+  issue: https://github.com/elastic/elasticsearch/issues/147199
+- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
+  method: test {p0=aggregations/terms_text_docvalues/Indexing disabled}
+  issue: https://github.com/elastic/elasticsearch/issues/147200
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=index/100_field_name_length_limit/Test field name length limit array}
   issue: https://github.com/elastic/elasticsearch/issues/147205

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -282,33 +282,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.MlMemoryIT
   method: testMemoryStats
   issue: https://github.com/elastic/elasticsearch/issues/147122
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
-  method: test {csv-spec:stats.statsWithSubstitutedExpressionOverFilters}
-  issue: https://github.com/elastic/elasticsearch/issues/147125
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
-  method: test {csv-spec:k8s-timeseries-arithmetic.division_metric_metric_grouping_filtering_ts}
-  issue: https://github.com/elastic/elasticsearch/issues/147126
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
-  method: test {csv-spec:bucket.docsBucketMonthlyHistogram}
-  issue: https://github.com/elastic/elasticsearch/issues/147127
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
-  method: test {csv-spec:stats.commonFilterExtractionWithAliasing}
-  issue: https://github.com/elastic/elasticsearch/issues/147128
-- class: org.elasticsearch.xpack.ccr.CcrSeqNoPruningIT
-  method: testSeqNoPrunedOnLeaderAfterFollowerCatchesUp
-  issue: https://github.com/elastic/elasticsearch/issues/147133
-- class: org.elasticsearch.xpack.esql.optimizer.ApproximationGoldenTests
-  method: testApproximationDisabled
-  issue: https://github.com/elastic/elasticsearch/issues/147144
-- class: org.elasticsearch.xpack.esql.plan.logical.CommandLicenseTests
-  method: testLicenseCheck
-  issue: https://github.com/elastic/elasticsearch/issues/147147
-- class: org.elasticsearch.ingest.common.IngestCommonClientYamlTestSuiteIT
-  method: test {yaml=ingest/70_bulk/Update with pipeline}
-  issue: https://github.com/elastic/elasticsearch/issues/147163
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/145901
 - class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
   method: testReplaceChildren {class org.elasticsearch.xpack.esql.plan.physical.ExchangeSinkExec}
   issue: https://github.com/elastic/elasticsearch/issues/147164
@@ -318,15 +291,6 @@ tests:
 - class: org.elasticsearch.xpack.security.operator.OperatorPrivilegesIT
   method: testNonOperatorUserCanCallAnalyzeRepositoryAPI
   issue: https://github.com/elastic/elasticsearch/issues/147171
-- class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
-  method: testReplaceChildren {class org.elasticsearch.xpack.esql.plan.logical.promql.operator.VectorBinaryComparison}
-  issue: https://github.com/elastic/elasticsearch/issues/147195
-- class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
-  method: testTransform {class org.elasticsearch.xpack.esql.plan.logical.promql.UnresolvedPromqlFunction}
-  issue: https://github.com/elastic/elasticsearch/issues/147199
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=aggregations/terms_text_docvalues/Indexing disabled}
-  issue: https://github.com/elastic/elasticsearch/issues/147200
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=index/100_field_name_length_limit/Test field name length limit array}
   issue: https://github.com/elastic/elasticsearch/issues/147205

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -282,9 +282,33 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.MlMemoryIT
   method: testMemoryStats
   issue: https://github.com/elastic/elasticsearch/issues/147122
-- class: org.elasticsearch.xpack.inference.action.filter.ShardBulkInferenceActionFilterIT
-  method: testBulkOperations {useLegacyFormat=false useSyntheticSource=false}
-  issue: https://github.com/elastic/elasticsearch/issues/147130
+- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
+  method: test {csv-spec:stats.statsWithSubstitutedExpressionOverFilters}
+  issue: https://github.com/elastic/elasticsearch/issues/147125
+- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
+  method: test {csv-spec:k8s-timeseries-arithmetic.division_metric_metric_grouping_filtering_ts}
+  issue: https://github.com/elastic/elasticsearch/issues/147126
+- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
+  method: test {csv-spec:bucket.docsBucketMonthlyHistogram}
+  issue: https://github.com/elastic/elasticsearch/issues/147127
+- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeApproximationIT
+  method: test {csv-spec:stats.commonFilterExtractionWithAliasing}
+  issue: https://github.com/elastic/elasticsearch/issues/147128
+- class: org.elasticsearch.xpack.ccr.CcrSeqNoPruningIT
+  method: testSeqNoPrunedOnLeaderAfterFollowerCatchesUp
+  issue: https://github.com/elastic/elasticsearch/issues/147133
+- class: org.elasticsearch.xpack.esql.optimizer.ApproximationGoldenTests
+  method: testApproximationDisabled
+  issue: https://github.com/elastic/elasticsearch/issues/147144
+- class: org.elasticsearch.xpack.esql.plan.logical.CommandLicenseTests
+  method: testLicenseCheck
+  issue: https://github.com/elastic/elasticsearch/issues/147147
+- class: org.elasticsearch.ingest.common.IngestCommonClientYamlTestSuiteIT
+  method: test {yaml=ingest/70_bulk/Update with pipeline}
+  issue: https://github.com/elastic/elasticsearch/issues/147163
+- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT
+  method: test
+  issue: https://github.com/elastic/elasticsearch/issues/145901
 - class: org.elasticsearch.xpack.esql.tree.EsqlNodeSubclassTests
   method: testReplaceChildren {class org.elasticsearch.xpack.esql.plan.physical.ExchangeSinkExec}
   issue: https://github.com/elastic/elasticsearch/issues/147164

